docs/api/cases/cases-api-create.asciidoc

```diff
@@ -34,6 +34,17 @@ default space is used.
 [role="child_attributes"]
 === {api-request-body-title}
 
+`assignees`::
+(Optional, array of objects) Array containing assignee objects.
++
+.Properties of assignee objects
+[%collapsible%open]
+=====
+`uid`:: 
+(Required, string) A user identifier to assign to the case.
+// TBD How are these values found?
```

Comment:
The profile ID is found by using Elasticsearch's suggest user profiles API: https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-suggest-user-profile.html

Reply:
Thanks!  I've mentioned those APIs in the descriptions via bb33f2329467aafd44fafbb3ca109ef1c91f527d